pages/public_cloud/public_cloud_network_services/ccm-octavia/guide.en-gb.md

```diff
@@ -0,0 +1,407 @@
+
+  Not supported when `lb-provider=ovn` is configured in openstack-cloud-controller-manager.
+
+- `loadbalancer.openstack.org/health-monitor-delay`
+
+  Defines the health monitor delay in seconds for the loadbalancer pools.
+
+- `loadbalancer.openstack.org/health-monitor-timeout`
+
+  Defines the health monitor timeout in seconds for the loadbalancer pools. This value should be less than delay
+
+- `loadbalancer.openstack.org/health-monitor-max-retries`
+
+  Defines the health monitor retry count for the loadbalancer pool members.
+
+- `loadbalancer.openstack.org/health-monitor-max-retries-down`
+
+  Defines the health monitor retry count for the loadbalancer pool members to be marked down.
+
+- `loadbalancer.openstack.org/flavor-id`
+
+  The id of the flavor that is used for creating the loadbalancer.
+
+  Not supported when `lb-provider=ovn` is configured in openstack-cloud-controller-manager.
+
+- `loadbalancer.openstack.org/availability-zone`
+
+  The name of the loadbalancer availability zone to use. It is ignored if the Octavia version doesn't support availability zones yet.
+
+  Not supported when `lb-provider=ovn` is configured in openstack-cloud-controller-manager.
+
+- `loadbalancer.openstack.org/default-tls-container-ref`
+
+  Reference to a tls container. This option works with Octavia, when this option is set then the cloud provider will create an Octavia Listener of type `TERMINATED_HTTPS` for a TLS Terminated loadbalancer.
+  Format for tls container ref: `https://{keymanager_host}/v1/containers/{uuid}`
+
+  When `container-store` parameter is set to `external` format for `default-tls-container-ref` could be any string.
+
+  Not supported when `lb-provider=ovn` is configured in openstack-cloud-controller-manager.
+
+- `loadbalancer.openstack.org/load-balancer-id`
+
+  This annotation is automatically added to the Service if it's not specified when creating. After the Service is created successfully it shouldn't be changed, otherwise the Service won't behave as expected.
+
+  If this annotation is specified with a valid cloud load balancer ID when creating Service, the Service is reusing this load balancer rather than creating another one. Again, it shouldn't be changed after the Service is created.
+
+  If this annotation is specified, the other annotations which define the load balancer features will be ignored.
+
+- `loadbalancer.openstack.org/hostname`
+
+  This annotations explicitly sets a hostname in the status of the load balancer service.
+
+- `loadbalancer.openstack.org/load-balancer-address`
+  
+  This annotation is automatically added and it contains the floating ip address of the load balancer service.
+  When using `loadbalancer.openstack.org/hostname` annotation it is the only place to see the real address of the load balancer.
+
+
+## Limitations
+
+- L7 and TLS Termination are not available yet.
+- You have to set a GatewayIP for your Subnet (we plan to perform it automatically in the future).
+
+> [!primary]
```

Comment:
not anymore, we should provide more details regarding the limitation.
If the customer already use a gateway that is not an openstack router and/or an OVHcloud managed Gateway, he should use a second subnet to host its load balancers using 'loadbalancerSubnetID'